assets/aml-benchmark/scripts/custom_dataset_preprocessors/truthfulqa_hf.py

```diff
@@ -0,0 +1,99 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ---------------------------------------------------------
+
+"""Custom preprocessor script for dataset:TruthfulQA, source:HF, config:generation."""
+
+from typing import Any, Dict, List, Union
+import argparse
+import json
+import logging as logger
+import pandas as pd
+
+
+def _parse_args():
+    """Parse the arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--input_path', type=str, required=True)
+    parser.add_argument('--output_path', type=str, required=True)
+    argss = parser.parse_args()
+    return argss
+
+
+def _read_jsonl_file(file_path: str) -> List[Dict[str, Any]]:
+    """Read `.jsonl` file and return a list of dictionaries."""
+    if not file_path.endswith(".jsonl"):
+        mssg = f"Input file '{file_path}' is not a .jsonl file."
+        logger.ERROR(mssg)
```

Comment:
We are already raising error in the line below. So we can remove this log statement.

Reply:
Removed. Thanks.